docs/sites.yml

```diff
@@ -7910,3 +7911,40 @@
     - Web Development
   built_by: IBM
   featured: true
+- title: Mevish Aslam, business coach
+  url: "https://mevishaslam.com/"
+  main_url: "https://mevishaslam.com/"
+  source_url: ""
```

Comment:
Please remove this field entirely if you don't want to provide url to source code / project is private

Reply:
I see, thanks for checking!